### DISCUSSION TITLE
Is DNS traffic always encrypted? What is the protocol used for non-DoH servers?

### DISCUSSION THREAD

**Discussion by sryze** [DNSCrypt/dnscrypt-proxy#2324] [Q&A] [answered]
I'm sorry if it's a dumb question, I've checked the wiki but still couldn't find the answer.

If I use the `google` resolver for example (from the list of public resolvers), will dnscrypt-proxy forward DNS requests using plain (unencrypted) DNS or some other protocol? I noticed that some servers in that list are marked as DoH and others are not, hence the question.

**Comment by jedisct1** [comment, marked as answer]
Hi!

With the exception of domains explicitly defined in `forwarding_rules` and bootstrap servers, `dnscrypt-proxy` will never send non-encrypted/non-authenticated traffic.

And the list of public resolvers only includes encrypted servers. Servers in that list that don't use DoH use the DNSCrypt protocol instead.
